core/Debian/stretch/Dockerfile

```diff
@@ -21,3 +22,6 @@ RUN apt-get update \
     && mkdir -p /etc/ansible \
     && echo "[local]" > /etc/ansible/hosts \
     && echo "localhost ansible_connection=local" >> /etc/ansible/hosts
+
+COPY davedaemon /usr/local/bin/
+RUN chmod 755 /usr/local/bin/davedaemon
```

Comment:
Well the original reasoning behind the calls to systemd/init were that without that systems it becomes impossible to test services. Since this was originally attempting to mirror an OS build, not a simple docker container but using Docker containers to do this.

There are obvious difficulties with this plan. Primarily that using init it is practically impossible to contain services to inside their containers(specifically agetty which can be spawned as root outside the container and spike up to 100% cpu), however SystemD is better in this regard as using the setting it keeps services running with the proper containment. These are the difficulties associated with the docker running on the host kernel and the privileges associated with PID 1. 

So I am tenuous, as with these changes this would be unable to test services similar to a tomcat role that hooks itself into the service system. 

Other ideas are expanding to have a set of containers that use a minimal docker cmd arguments or a mini-service manage such as what you have done here so tests can be tailored to the environment as java or other binary installations generally don't require the heavy-handed service manager that is used currently in ansiblecheck.

What are your/general opinions, as I would like this to tailor generally? 

As you can see from the above there are some serious technical limitations I have been unable to engineer around at this point.